content/en/agent/guide/private-link.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How to setup a PrivateLink to Datadog?
+kind: guide
+further_reading:
+- link: "agent/logs"
+  tag: "Documentation"
+  text: "Enable log collection with the Agent."
+- link: "/integrations/amazon_web_services/#set-up-the-datadog-lambda-function"
+  tag: "Documentation"
+  text: "Collect logs from your AWS services"
+---
+
+<div class="alert alert-warning">
+The PrivateLink setup is only available to <b>forward Logs to Datadog in AWS us-east-1 region</b>. Reach out to <a href="/help">Datadog Support</a> to enable this feature for your organization.
+</div>
+
+The goal of this guide is to explain how to configure a PrviateLink for a given VPC. This allows you to send your log traffic to Datadog without going over the public internet, thereby reducing transit costs and improving security.
+
+## Overview
+
+Datadog provides a VPC endpoint service which lives in the AWS `us-east-1` region.
+
+The overall process consist in configuring an internal VPC endpoint that local Datadog Agents can send data to. This VPC endpoint is then peered with the Datadog VPC endpoint service.
+
+{{< img src="agent/guide/private_link/vpc_schema.png" alt="VPC Schema" >}}
```

Comment:
Can we work with design on better images here?  Also lets avoid abbreviating Datadog to DD.

Reply:
Yup, already open a request to design for this.